express-post-json/server.ts

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+
+type Grade = {
+  id: number;
+  name: string;
+  course: string;
+  score: number;
+};
+
+type NextId = number;
+
+const grades: Record<number, Grade> = {};
+
+let nextId: NextId = 1;
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  const localArr: Grade[] = Object.values(grades);
+  res.json(localArr);
+});
+
+app.use(express.json());
```

Comment:
We usually put this at the top, before all the routes, just in case any of the routes need it.

Reply:
is adding it at the top giving the ability to parse all JSON requests, no matter the method?